transport/awssqs/consumer.go

```diff
@@ -0,0 +1,242 @@
+package awssqs
```

Comment:
Is there any other `sqs` except `awssqs`? Could this be `sqs`? Or maybe that conflicts with the package name of the SDK?

Reply:
Indeed this was done to avoid conflicts with the SDK, and to follow the naming for awslambda. I can change it to sqs if you want me to, conflicts with the SDK can be solved by adding an alias when needed.